plugin/helpers_bots.go

```diff
@@ -9,6 +9,25 @@ import (
 	"github.com/pkg/errors"
 )
 
+type Options struct {
```

Comment:
@alxsah surfaced this in a discussion https://community-daily.mattermost.com/core/pl/syptqqqgejn4pnm516qspyohmw with respect to issues generating our mockery stubs.

Reposting my thoughts here:
> I confess I'd personally prefer to see ensureBotOptions and ensureBotOption exported instead of trying to force the developer to only use the given helper functions. There's a decent chance someone might want to refactor a variant that we don't think of, and it seems reasonable to allow them to do so.

The usual argument for not exporting this is to avoid someone configuring a "bad" options, but I don't think there's a good way to do that in the options framework anyway, so we can just fail on the `EnsureBot` call if you specify bad options. 

I'm personally inclined to make all parameters and return values of exported functions themselves exported as well.

Reply:
I'll go with @lieut-data's suggestion then and export `ensureBotOptions` and `ensureBotOption`.